assets/js/app.js

```diff
@@ -394,4 +399,20 @@ class App {
             });
         });
     }
+
+    #registerMercureUpdateListener() {
+        const mercureURL = document.body.getAttribute('data-ea-mercure-url');
+        if (mercureURL.substring(0,4) != 'http') return;
```

Comment:
```suggestion
        if (mercureURL.substring(0,4) !== 'http') return;
```

But why is this check necessary? A relative URL should work.